$NetBSD: patch-platform_media_libjpeg_moz.build,v 1.1 2024/09/17 06:24:18 martin Exp $

Hard-code compiler flags for altivec support to -maltivec
instead of the undefined config list PPC_VMX_FLAGS.

This matches other places where -maltivec is used, like:
	platform/gfx/cairo/libpixman/src/moz.build
	platform/gfx/qcms/moz.build

--- platform/media/libjpeg/moz.build.orig	2024-09-05 10:13:09.000000000 +0200
+++ platform/media/libjpeg/moz.build	2024-09-15 20:36:37.420052767 +0200
@@ -240,7 +240,7 @@
         'simd/powerpc/jsimd.c',
     ]
     for srcfile in ppc_vmx_sources:
-        SOURCES[srcfile].flags += CONFIG['PPC_VMX_FLAGS']
+        SOURCES[srcfile].flags += ['-maltivec']
 else: # No SIMD support?
     SOURCES += [
         'jsimd_none.c',
